SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialLoadBalanceEnforcer.h

```diff
@@ -19,20 +19,28 @@ class SpatialLoadBalanceEnforcer
 	struct AclWriteAuthorityRequest
```

Comment:
This struct now has full information on how to update the ACL component, so can be treated as a message that is passed to the sender which understands what to do entirely from the state of the message